### PR TITLE
Humble Feb + Some other games

### PR DESCRIPTION
**Pull request by daddeltrotter**

### CODE REVIEW

File: tweaks/steam.yaml [hunk 1]
```diff
@@ -1318,6 +1357,9 @@
 "798790": # PolyCube
   compat_tool: proton_513
 
+"349730": Popup Dungeon
```
Comment (alkazar) [inline tweaks/steam.yaml:1360]: missing '#' :sweat_smile:

Comment (daddeltrotter) [reply tweaks/steam.yaml]: Done. These hashes are driving me crazy.
Any chance that we create a test automate who validates the entries?

Comment (alkazar) [reply tweaks/steam.yaml]: Yeah, I have been thinking about that as well.

Comment (alkazar) [reply tweaks/steam.yaml]: I usually run this simple python script, but it doesn't catch everything
```
import yaml
data = yaml.load(open('./tweaks/steam.yaml'), Loader=yaml.FullLoader)
```

**Pull request merged**